docs/content/testnet/rvasps.en.md

```diff
@@ -101,26 +103,76 @@ To use the protocol buffers directly, use the `TRISAIntegration` service `Transf
+
+##### SendError
+
+The `SendError` policy exists to test handling for error envelopes. In this policy, the rVASP will simply send an error envelope with the `ComplianceCheckFail` TRISA error code.
+
+### Sending a TRISA message to an rVASP
+
+The rVASP expects a `trisa.data.generic.v1beta1.Transaction` as the transaction payload and an `ivms101.IdentityPayload` as the identity payload. The transaction payload must contain a beneficiary address that matches the rVASP beneficiaries from the table above; e.g. use:
 
 ```json
 {
-    "txid": "1234",
     "originator": "anydatahereisfine",
     "beneficiary": "1MRCxvEpBoY8qajrmNTSrcfXSZ2wsrGeha",
     "amount": 0.3,
-    "network": "TestNet",
-    "timestamp": "2021-06-14T16:41:52-05:00"
+    "network": "TestNet"
 }
 ```
 
-You may specify any `txid` or `originator` string and the `network` and `timestamp` fields are ignored.
+You may specify any `originator` "wallet address" string and the `network` and `asset_type` fields are ignored if you would like to simulate other transaction information.
+
+The identity payload must also be not null and must be valid IVMS101, but may be partially or completely filled in depending on the corresponding beneficiary policy.
+
+- Partial Identity: Contains only the originator and originating vasp identities
+- Complete Identity: Contains the originator, benefciiary, and originating and beneficiary vasp identities.
 
 Create a sealed envelope either using the directory service or direct key exchange to fetch the rVASP RSA public keys and using `AES256-GCM` and `HMAC-SHA256` as the envelope cryptography. Then use the `TRISANetwork` service `Transfer` RPC to send the sealed envelope to the rVASP.
 
-TODO: Soon the `trisa` command line program will be available. We will specify how to use the CLI program to send a message here as soon as it is released.
+See [Secure Envelopes]({{< ref "secure-envelopes" >}}) for more on how to compose a valid secure envelope for transfers and the [TRISA CLI]({{< ref "testnet/trisa-cli" >}}) for more on using a command line application for sending transfers.
+
+#### Beneficiary Policies
+
+The _beneficiary policy_ determines how an rVASP responds to an incoming TRISA transfer message to its TRISA endpoint. Beneficiary policies are mapped to wallet addresses, and the rVASP identifies the beneficiary address using the wallet address in the transaction payload. A description of the beneficiary policies follow.
+
+##### SyncRepair
+
+For the `SyncRepair` policy, the identity payload does not have to include the beneficiary identity, although it must be not null. The rVASP will respond synchronously with a response envelope containing a `received_at` timestamp and the complete beneficiary identity.
```

Comment:
Will respond synchronously meaning it will "accept" the transaction; echoing the identity and transaction and including the `received_at` timestamp.